compiler/noirc_frontend/src/parser/errors.rs

```diff
@@ -78,6 +78,12 @@ pub enum ParserErrorReason {
     TraitImplVisibilityIgnored,
     #[error("comptime keyword is deprecated")]
     ComptimeDeprecated,
+    #[error(
+        "Only one 'cfg' attribute is allowed, but found {cfg_attribute} and {second_cfg_attribute}"
+    )]
+    MultipleCfgAttributesFound { cfg_attribute: CfgAttribute, second_cfg_attribute: CfgAttribute },
```

Comment:
I think Rust allows multiple config attributes

Reply:
Yes, Rust allows e.g. duplicate config attributes:
```rust
#[cfg(feature="default")]
#[cfg(feature="default")]
fn main() {
    println!("Hello, world!");
}
```

but my current plan is to support this using `any`/`all`/etc., so it could be addressed in a follow up.

Do you have a use case for multiple config attributes?

Reply:
No use case but I saw it working in Rust, so... For example you could have a `cfg(test)` and `cfg(feature = "foo")` on top of a test, though if there's going to be `all` and `any` then it's not necessary.